src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasClient.kt

```diff
@@ -21,30 +21,31 @@ import com.xenomachina.argparser.ArgParser
 import com.xenomachina.argparser.default
 import io.grpc.ManagedChannel
 import io.grpc.ManagedChannelBuilder
+import org.slf4j.LoggerFactory
 import java.util.concurrent.TimeUnit
 
 /**
  * Simple client that will send a request to Cosmas server and wait for response
  * @author Aleksandr Fedotov (iisuslik43)
  */
 class CosmasClient(host: String, port: Int) {
-
+    private val log = LoggerFactory.getLogger(this::class.java)

```

Comment:
This looks a bit dangerous because moving this code will change the effective logger name as well. Normally one would use explicit class reference (CosmasClient::class.java) or even strings (e.g. "CosmasClient") to avoid long logger names.

Also, please capitalize `log` and even move it outside of the class, so that it could be used without `this.` prefix and it was clear it is a logger, not some local var:
`LOG.info`

Reply:
But if I move `log` outside and do it with all files in package, it will be conflicting declarations, so maybe I should make it static.

Reply:
If you make them private val you will have no conflicts.
I just've googled kotlin logging and oh dear how many interesting solutions are out there.